webapp/src/routes/game/[gameId]/+page.svelte

```diff
@@ -4,18 +4,20 @@
 	import { onMount } from 'svelte';
 	import { supabase } from '$lib/supabaseClient';
 	import { error } from '@sveltejs/kit';
-	import Toasts from '$lib/components/alert/Toasts.svelte';
 	import { addToast } from '$lib/stores/toastStore';
+	import { Confetti } from 'svelte-confetti';
+	import { goto } from '$app/navigation';
+	import Toasts from '$lib/components/alert/Toasts.svelte';
 	import CardTable from '$lib/components/CardTable.svelte';
 	import PlayerDeck from '$lib/components/PlayerDeck.svelte';
 	import PlayerSelfDeck from '$lib/components/PlayerSelfDeck.svelte';
 	import EmojiClick from '$lib/components/EmojiClick.svelte';
 	import FlyingPlayCards from '$lib/components/FlyingPlayCards.svelte';
-	import { Confetti } from 'svelte-confetti';
 	import LoadingBar from '$lib/components/LoadingBar.svelte';
+	import GameEndScreen from '$lib/components/GameEndScreen.svelte';
 
 	export let data: PageData;
-	let myPlayerId: string | null = null;
+	let storedPlayerId: string | null = null;
 	let myPlayer: Player | null = null;
```

Comment:
Also change this name then. Or delete if not used.

Reply:
Not sure what you mean. I just wanted to separate the variable name from data.player[_something_].id and myPlayer.id because a storedPlayerId could also be invalid if created from a different game.

Reply:
I just meant that the "myPlayer" variable is obsolete as well then. Or is it used somewhere?

Reply:
Actually we use it all over the site for different things lol. It should stay 😅

Reply:
then why not also call it storedPlayer?

Reply:
because the data for myPlayer comes from the database and the id inside storedPlayerId comes from the local Storage. The data  for myPlayer comes from the database can be used safely because it is always valid. Data from the local Storage could be invalid. I wanted to make the difference clear. because the data for myPlayer comes from the database and the id in storedPlayerId comes from local storage. The data for myPlayer that comes from the database can be used safely because it is always valid. The data from local storage can be invalid and should only be used for validation (actually we do, but it is fine and the work to change it is not worth it). I wanted to make the difference clear. - If the data from myPlayer would come from the local Storage it should be called storedmyPlayer, but it doesnt.because the data for myPlayer comes from the database and the id in storedPlayerId comes from local storage. The data for myPlayer that comes from the database can be used safely because it is always valid. The data from the local storage might be invalid. I wanted to make the difference clear because the data for myPlayer comes from the database and the id in storedPlayerId comes from local storage. The data for myPlayer that comes from the database can be used safely because it is always valid. The data from local storage can be invalid and should only be used for validation (actually we do, but it is fine and the work to change it is not worth it). I wanted to make the difference clear. - If the myPlayer data came from the local storage, it should be called storedmyPlayer, but it does not.